sales/send_quotations/get_signature_to_validate.rst

```diff
@@ -2,31 +2,51 @@
 Get a signature to confirm an order
 ===================================
 
-You can use online signature to get orders automatically confirmed. Both
-you and your customer will save time by using this feature compared to a
-traditional process.
+Online signatures are like electronic "fingerprints". With their use on Odoo, you will get
+automatically confirmed orders. Indeed, you and your customers will save a lot of time by using this
+feature compared to a traditional process.
 
-Activate online signature
-=========================
+How to enable online signature
+==============================
 
-Go to :menuselection:`SALES --> Configuration --> Settings` and activate
-the *Online Signature & Payment* feature.
+Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Online Signature**
+feature.
 
-.. image:: media/getsignaturetovalidate01.png
-    :align: center 
+.. image:: media/signature_1.png
+   :align: center
+   :class: img-thumbnail
+   :alt: How to enable online signature on Odoo Sales?
 
 .. note::
-   If you are using :doc:`quotation templates <quote_template>`, you can also pick a default setting
-   for each template.
+   If you are familiar with this documentation: :doc:`quote_template`, you can activate or not the
+   **Online Signature** feature for each quotation template you use, under their confirmation tab.
+   Example:
```

Comment:
Technical nitpicking: I would leave a blank line before this one. Even if the render looks nice when you build it now, it may not stay that way if it happens that the layout is changed.